README.rst

```diff
@@ -261,9 +261,7 @@ Usage
 function locally, and it’s also true for spawning API Gateway locally -
 If no template is specified ``template.yaml`` will be used instead.
 
-You can find sample SAM templates either under **``samples``** located
-in this repo or by visiting
-`SAM <https://github.com/awslabs/serverless-application-model>`__
+You can find sample SAM templates by visiting `SAM <https://github.com/awslabs/serverless-application-model>`__
```

Comment:
Can you add a note here about the `sam init` command. Something to the effect:

"You can create a sample app by running the command `sam init --runtime <your-favorite-runtime>` or find other sample SAM Templates by visiting `SAM <https://github.com/awslabs/serverless-application-model>`__ official repository."

Reply:
Thanks for your review.
I fixed the message. Would you review this again?